src/solvers/MFEMSuperLU.C

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include "MFEMSuperLU.h"
+#include "MFEMProblem.h"
+
+registerMooseObject("PlatypusApp", MFEMSuperLU);
+
+InputParameters
+MFEMSuperLU::validParams()
+{
+  InputParameters params = MFEMSolverBase::validParams();
```

Comment:
The `mfem::SuperLUSolver` class has a [number of solver options](https://docs.mfem.org/html/classmfem_1_1SuperLUSolver.html) which (maybe?) could be useful for the user to have the ability to set.

Reply:
Agreed - but I'd somewhat be keen to extend the solver options in a later PR so they can be properly tested without blocking the key feature in this one (of adding the ability to set preconditioners and solvers at all)

Reply:
That's fair enough! We can put a pin on this to come back to it later